man/systemctl.xml

```diff
@@ -570,12 +570,11 @@
         <term><option>--runtime</option></term>
 
         <listitem>
-          <para>When used with <command>enable</command>,
-          <command>disable</command>, <command>edit</command>,
+          <para>When used with <command>enable</command> and <command>edit</command>,
           (and related commands), make changes only temporarily, so
           that they are lost on the next reboot. This will have the
           effect that changes are not made in subdirectories of
-          <filename>/etc</filename> but in <filename>/run</filename>,
+          <filename>/etc</filename>, but in <filename>/run</filename>,
           with identical immediate effects, however, since the latter
           is lost on reboot, the changes are lost too.</para>
```

Comment:
it might make sense to also document positively that `disable` removes from both directories...